src/ir/cost.h

```diff
@@ -537,6 +537,7 @@ struct CostAnalyzer : public OverriddenVisitor<CostAnalyzer, Index> {
   Index visitSIMDShuffle(SIMDShuffle* curr) {
     return 1 + visit(curr->left) + visit(curr->right);
   }
+  Index visitPrefetch(Prefetch* curr) { return 1 + visit(curr->ptr); }
```

Comment:
I doubt it matters, but maybe this could be 0 instead of 1, as it is meant to speed up later things?